testing/cucumber/features/linters/scenario_with_bad_name.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Scenario with bad name linter
+
+  As a reader of documentation
+  I want every scenario to have Given and When steps rather than Test, Verify or Check
+  So that the naming is consistent and intent clear
+
```

Comment:
The linter is about the name of a test, not its steps. Although I realize that this is based on the original message provided by GherkinLint, I have no idea why steps were mentioned back then either.

The use of "intent" I agree with but I've seen enough test suites in the wild to know that 'consistent' and 'bad' are not mutually exclusive. ;)

The original summary from GherkinLint wasn't too bad, so perhaps we could tweak it to be something like this?

```
  As a writer of documentation
  I want to be warned about 'assertion' style scenario names
  So that scenarios don't sound like tests
```